src/components/DynamicFormContainer/DynamicFormLayout.test.tsx

```diff
@@ -96,21 +66,33 @@ const whenIsTransferableRecord = (): void => {
       templateIndex: 0,
       ownership: { holderAddress: "", beneficiaryAddress: "" },
     },
+    currentFormTemplate: {
+      name: "Bill of Lading",
+      type: "TRANSFERABLE_RECORD",
+      defaults: {},
+      schema: {
+        type: "object",
+        properties: {
+          foo: { type: "string", title: "Foo Field" },
+          bar: { type: "string" },
+        },
+      },
+    },
   });
 };
 
 const whenActiveFormIndexIsNotAvailable = (): void => {
-  mockUseConfigContext.mockReturnValue({ config: sampleConfig });
   mockUseFormsContext.mockReturnValue({
     activeFormIndex: undefined,
     setActiveFormIndex: mockSetActiveFormIndex,
+    currentFormTemplate: undefined,
```

Comment:
why is this needed ?

Reply:
its not needed, was just being more explicit